TMP2022/technical-writing-track/precious-tw-funke/README.md

```diff
@@ -0,0 +1,239 @@
+# Kuda Bank User Guide For Android Users
+
+Nigeria's first mobile-only bank, [Kuda Bank](https://kuda.com/), was granted a license by the Central Bank of Nigeria (CBN) in 2019. Consequently, Kuda is only available online and has no physical locations. Babs Ogundeyi and Musty Mustapha formed Kuda Bank. This guide illustrates getting started with the Kuda bank app.
```

Comment:
Babs Ogundeyi and Musty Mustapha founded Kuda Bank.

Reply:
Corrected.